python/lzreposync/src/lzreposync/deb_repo.py

```diff
@@ -0,0 +1,236 @@
+#  pylint: disable=missing-module-docstring
+
+import logging
+import os
+import time
+import urllib.parse as urlparse  # pylint: disable=F0401,E0611
+from shutil import copyfile
+from urllib.parse import unquote
+
+import requests
+
+from lzreposync.deb_metadata_parser import DEBMetadataParser
+from lzreposync.packages_parser import PackagesParser
+from lzreposync.repo import Repo
+from lzreposync.translation_parser import TranslationParser
+from spacewalk.satellite_tools.syncLib import log2
+from spacewalk.server import rhnSQL
+from uyuni.common import fileutils
+
+RETRIES = 10
+RETRY_DELAY = 1
+FORMAT_PRIORITY = [".xz", ".gz", ""]
+log = logging.getLogger(__name__)
+
+
+#  pylint: disable-next=missing-class-docstring
+class DebRepo(Repo):
```

Comment:
We should consolidate the debian repo parsing. Right now it's split into `repo_plugins.deb_src`, `common.repo` and this `deb_repo` (which appears to be a copy of the class in `deb_src`